packages/atomic/src/utils/combobox.tsx

```diff
@@ -190,19 +190,18 @@ export class Combobox {
       'aria-owns': `${this.options.id}-listbox`,
       role: 'combobox',
       'aria-haspopup': 'listbox',
+      'aria-expanded': `${this.hasValues}`,
```

Comment:
I can confirm that `cypress-axe` works because it found this accessibility issue.

Reply:
💯